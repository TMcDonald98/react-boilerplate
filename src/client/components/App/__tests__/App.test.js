import * as React from 'react';
import {
  App,
  // mapStateToProps
} from '../App';

describe('components/App', () => {
  describe('render', () => {
    it('should render the component', () => {
      const result = shallow(<App />);

      expect(result).toMatchSnapshot('App');
    });
  });
});
