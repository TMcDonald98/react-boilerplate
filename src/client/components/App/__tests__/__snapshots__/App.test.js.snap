// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/App render should render the component: App 1`] = `
<div
  className="undefined app-container"
>
  <div
    className="content-container"
  >
    <Connect(Switch)
      routes={
        Object {
          "@@redux-first-router/NOT_FOUND": [Function],
          "HOME": [Function],
        }
      }
    />
  </div>
</div>
`;
